Packs/CortexXDR/Playbooks/playbook-Cortex_XDR_Lite_-_Incident_Handling_README.md

```diff
@@ -0,0 +1,75 @@
+---
+
+## Cortex XDR Lite - Incident Handling
+
+The **Cortex XDR Lite - Incident Handling** playbook is triggered by fetching a Palo Alto Networks Cortex XDR incident and executes the following:
+
+**Analysis:**
+- Enriches all the indicators in the incidents.
+
+**Investigation:**
+- Checks for related XDR alerts to the user and the endpoint by Mitre tactics.
+- Checks for specific arguments for malicious usage from the command line.
+
+**Verdict:**
+- Determines the appropriate verdict based on the analysis and the investigation findings.
+
+**Verdict Handling:**
+- Handles malicious incidents by initiating appropriate response actions.
```

Comment:
add the actions we are doing for each entity (host, user, indicators)

Reply:
Done